adapter/src/components/Alerts.js

```diff
@@ -25,8 +25,8 @@ const Alerts = () => {
         )
 
     useEffect(() => {
-        if (alertManagerAlerts.length > 0 || alertStackAlerts.length > 0) {
-            setAlertStackAlerts(
+        if (alertManagerAlerts.length > 0) {
+            setAlertStackAlerts((alertStackAlerts) =>
```

Comment:
Very minor nitpick: This variable has the same name as the variable returned by `useState` in the outer scope. Maybe `currentAlertStackAlert` (or another name that ensures that the reader immediately recognizes that this is a different variable)?

Reply:
Yeah that's a nice readability improvement 👍